src/keria/app/agenting.py

```diff
@@ -865,20 +1105,28 @@ def recur(self, tyme=None):
 
 
 class Initer(doing.Doer):
+    """Prints a message once an agent is initialized."""
     def __init__(self, agentHab, caid, tock=0.0):
         self.agentHab = agentHab
         self.caid = caid
         self.tock = tock
         super(Initer, self).__init__(tock=self.tock)
 
-    def recur(self, tyme):
-        """ Prints Agent name and prefix """
+    def print_agent(self):
```

Comment:
This does not follow the camelCase naming convention.

Reply:
I did not change this in the PR, just the spacing changed.